diff --git a/dist/commands/options.js b/dist/commands/options.js
index e8ae40a89c57b10d956ff8677a34f131b1c2cbb8..db387c099fe0ac727d3091bc5035c11c3000967c 100644
--- a/dist/commands/options.js
+++ b/dist/commands/options.js
@@ -93,10 +93,10 @@ const bundleCommandOptions = exports.bundleCommandOptions = [{
   description: '(unsupported) Resets the transformation cache'
 },
 // noop, needed for compatibility
-{
-  name: '--config-cmd',
-  description: '(unsupported) Command to generate a JSON project config'
-},
+// {
+//   name: '--config-cmd',
+//   description: '(unsupported) Command to generate a JSON project config'
+// },
 // options specific to Re.Pack
 {
   name: '--json <statsFile>',
@@ -105,10 +105,12 @@ const bundleCommandOptions = exports.bundleCommandOptions = [{
 }, {
   name: '--stats <preset>',
   description: 'It instructs Webpack on how to treat the stats:\n' + "'errors-only'  - only output when errors happen\n" + "'errors-warnings' - only output errors and warnings happen\n" + "'minimal' - only output when errors or new compilation happen\n" + "'none' - output nothing\n" + "'normal' - standard output\n" + "'verbose' - output everything\n" + "'detailed' - output everything except chunkModules and chunkRootModules\n" + "'summary' - output webpack version, warnings count and errors count"
-}, {
-  name: '--verbose',
-  description: 'Enables verbose logging'
-}, {
+},
+// {
+//   name: '--verbose',
+//   description: 'Enables verbose logging'
+// },
+{
   name: '--watch',
   description: 'Watch for file changes'
 }, {
