diff --git a/scripts/react-native-xcode.sh b/scripts/react-native-xcode.sh
index b5d50ccb4bb4967edb575995ba727acce99dfd4a..7ad7e79f4e58ac388cb05f58535c1ed8f77a37da 100755
--- a/scripts/react-native-xcode.sh
+++ b/scripts/react-native-xcode.sh
@@ -90,7 +90,7 @@ fi
 
 [ -z "$NODE_ARGS" ] && export NODE_ARGS=""
 
-[ -z "$CLI_PATH" ] && CLI_PATH="$REACT_NATIVE_DIR/scripts/bundle.js"
+[ -z "$CLI_PATH" ] && CLI_PATH="$REACT_NATIVE_DIR/cli.js"
 
 [ -z "$BUNDLE_COMMAND" ] && BUNDLE_COMMAND="bundle"
 
